src/styles/utiles/_variables.scss

```
@@ -0,0 +1,11 @@
+$cell-size: 100px;
+$day-in-month: 31;
+$cell-gap: 1px;
+$cell-shift: 
+  "mon" 0,
+  "tue" ($cell-size + $cell-gap),
+  "wed" (2 *($cell-size + $cell-gap)),
+  "thu" (3 *($cell-size + $cell-gap)),
+  "fri" (4 *($cell-size + $cell-gap)),
+  "sat" (5 *($cell-size + $cell-gap)),
+  "sun" (6 *($cell-size + $cell_gap));
```

calculate margin inside @each

```suggestion
$cell-shift: 
  "mon" 0,
  "tue" 1,
  "wed" 2,
  "thu" 3,
  "fri" 4,
  "sat" 5,
  "sun" 6;
```